### ISSUE TITLE
Azure Deployment

### ISSUE THREAD

**Issue by francoisdutoit** [fullstackhero/blazor-starter-kit#225]
Hi - Are there any configuration changes that need to be made to deploy to Azure App Service?

I assume we only have to publish the server app?

Below is the error that I'm getting in the Azure logs....

Application: w3wp.exe
CoreCLR Version: 5.0.721.25508
.NET Version: 5.0.7
Description: The process was terminated due to an unhandled exception.
Exception Info: System.IO.DirectoryNotFoundException: C:\home\site\wwwroot\Files\
   at Microsoft.Extensions.FileProviders.PhysicalFileProvider..ctor(String root, ExclusionFilters filters)
   at Microsoft.Extensions.FileProviders.PhysicalFileProvider..ctor(String root)
   at Contracon.Server.Startup.Configure(IApplicationBuilder app, IWebHostEnvironment env, IStringLocalizer`1 localizer) in C:\repositories\Contracon.Clean\ContraconWeb\Contracon\src\Server\Startup.cs:line 78
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at Microsoft.AspNetCore.Hosting.ConfigureBuilder.Invoke(Object instance, IApplicationBuilder builder)
   at Microsoft.AspNetCore.Hosting.ConfigureBuilder.<>c__DisplayClass4_0.<Build>b__0(IApplicationBuilder builder)
   at Microsoft.AspNetCore.Hosting.GenericWebHostBuilder.<>c__DisplayClass15_0.<UseStartup>b__1(IApplicationBuilder app)
   at Microsoft.Extensions.DependencyInjection.AutoRegisterMiddleware.<>c__DisplayClass4_0.<Configure>b__0(IApplicationBuilder app)
   at Microsoft.AspNetCore.Mvc.Filters.MiddlewareFilterBuilderStartupFilter.<>c__DisplayClass0_0.<Configure>g__MiddlewareFilterBuilder|0(IApplicationBuilder builder)
   at Microsoft.AspNetCore.Server.IIS.Core.IISServerSetupFilter.<>c__DisplayClass2_0.<Configure>b__0(IApplicationBuilder app)
   at Microsoft.AspNetCore.HostFilteringStartupFilter.<>c__DisplayClass0_0.<Configure>b__0(IApplicationBuilder app)
   at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)
   at Microsoft.Extensions.Hosting.Internal.Host.StartAsync(CancellationToken cancellationToken)
   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.RunAsync(IHost host, CancellationToken token)
   at Microsoft.Extensions.Hosting.HostingAbstractionsHostExtensions.RunAsync(IHost host, CancellationToken token)
   at Contracon.Server.Program.Main(String[] args) in C:\repositories\Contracon.Clean\ContraconWeb\Contracon\src\Server\Program.cs:line 42
   at Contracon.Server.Program.<Main>(String[] args)

Thanks!

**Comment by iammukeshm** [comment]
As the error suggests, can you create a new folder "Files" under the wwwroot folder once it's published? I think that should solve your issue.

**Comment by francoisdutoit** [comment]
@iammukeshm 

Hi Mukesh,

Thanks for the feedback. I resolved the issue by updating the files in the below directories to "Copy if newer" - they were set to "Do not copy" which meant that the folders were not created on Azure.

src\Server\Files\Images\Products\demo.jpg  
src\Server\Files\Documents\dummy.txt

The next issue I faced was the below...

System.Globalization.CultureNotFoundException: Culture is not supported. (Parameter 'name') km_KH is an invalid culture identifier.

I just commented out the Khmer language code in src\Shared\Constants\Localization\LocalizationConstants.cs 

After that, the Azure app service deployment works fine :)

Thanks!
